src/main/java/org/sonar/report/pdf/builder/RuleBuilder.java

```diff
@@ -93,7 +93,10 @@ public void loadViolatedResources(final Rule rule, final String ruleKey, final S
       // ruleKey = URLEncoder.encode(ruleKey, "UTF8");
       LOG.debug("Accessing Sonar: getting violated resurces by one given rule (" + ruleKey + ")");
 
-      SonarClient client = SonarClient.create(credentials.getUrl());
+      SonarClient client = SonarClient.builder().url(credentials.getUrl())
+                                                .login(credentials.getUsername())
```

Comment:
I'm using the PDF-Report in version 4.5.4 and it needs the values for user and password to be passed as -D options on the runner to execute properly. It's not even bothering to read the values we pass on the settings page for the plugin.